manifests/0000_50_cluster-platform-operator-manager_00-rukpak-bundles.crd.yaml

```diff
@@ -37,10 +38,14 @@ spec:
         description: Bundle is the Schema for the bundles API
         properties:
           apiVersion:
-            description: 'APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            description: 'APIVersion defines the versioned schema of this representation
```

Comment:
Just curious, did `make manifests` do this? It is weird it hasn't done it before.

Reply:
I assume so? I didn't open any of those files myself 🤷‍♀️